engine/installation/linux/ubuntulinux.md

```diff
@@ -116,365 +117,430 @@ packages from the Docker repository:
               500 https://apt.dockerproject.org/repo/ ubuntu-trusty/main amd64 Packages
            1.12.0-0~trusty 0
               500 https://apt.dockerproject.org/repo/ ubuntu-trusty/main amd64 Packages
-           1.11.2-0~trusty 0
-              500 https://apt.dockerproject.org/repo/ ubuntu-trusty/main amd64 Packages
-           1.11.1-0~trusty 0
-              500 https://apt.dockerproject.org/repo/ ubuntu-trusty/main amd64 Packages
     ```
 From now on when you run `apt-get upgrade`, `APT` pulls from the new repository.
 
 ### Prerequisites by Ubuntu Version
 
-- Ubuntu Xenial 16.04 (LTS)
-- Ubuntu Wily 15.10
-- Ubuntu Trusty 14.04 (LTS)
+#### Ubuntu Xenial 16.04 (LTS), Wily 15.10, Trusty 14.04 (LTS)
```

Comment:
Note that this will conflict with the `vnext-engine` branch; Support for Ubuntu Wily 15.10 has been removed for Docker 1.13, because it's EOL.

Perhaps we should mention that it's recommended to run an LTS version, or at least make users aware that non-LTS versions only have a 9-month life-span, after which _no_ updates, including security updates are provided by Ubuntu.